### PR TITLE
switch all of get-pipsi.py to subpocess usage and prepare for win32

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
it has a bug, will get fixed in the full windows port

File: get-pipsi.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 import os
 import sys
+from subprocess import call
```
Comment (untitaker) [inline get-pipsi.py]: As said, you need to wrap this for all invocations of ``call``, and catch OSError:

- Windows raises [WindowsError](https://docs.python.org/2/library/exceptions.html?highlight=windowserror#exceptions.WindowsError) when commands are not found, which is a subclass of ``OSError``.
- Python 2 raises OSError, Python 3 raises FileNotFoundError.

File: get-pipsi.py [hunk 2]
```diff
@@ -47,17 +52,17 @@ def _cleanup():
         except (OSError, IOError):
             pass
 
-    if Popen(['virtualenv', venv]).wait() != 0:
+    if call(['virtualenv', venv]) != 0:
         _cleanup()
         fail('Could not create virtualenv for pipsi :(')
 
-    if Popen([venv + '/bin/pip', 'install', 'pipsi']).wait() != 0:
+    if call([venv + '/bin/pip', 'install', 'pipsi']) != 0:
```
Comment (untitaker) [inline get-pipsi.py]: This breaks under Windows. Windows has ``Scripts``, ``Include``, and ``Lib`` as subfolders.

**Comment by RonnyPfannschmidt** [comment]
closing this for a new one with full windows support

**Pull request closed without merging**